.pre-commit-config.yaml

```diff
@@ -1,21 +1,25 @@
+fail_fast: true
```

Comment:
With python tooling, there are some issues. Isort configuration isn't used on pipelines and causes couple configuration mismatch issues. Needs to be fixed in another pull-request.

Also, we have husky and pre-commit on top of each other. I am uncertain if this is the best solution.